### PR TITLE
OpenBSD support and slackware build instructions

### PR DESCRIPTION
**Pull request by glowiak**
title

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -54,7 +55,7 @@ This is the preferred method for installation, and is suitable for packages.
 # configure everything
 cmake -S . -B build \
    -DCMAKE_BUILD_TYPE=Release \
-   -DCMAKE_INSTALL_PREFIX="/usr" \ # Use "/usr" for packages, otherwise, leave it at the default "/usr/local".
+   -DCMAKE_INSTALL_PREFIX="/usr" \ # Replace /usr with /usr/local when building on FreeBSD
```
Comment (ghost) [inline BUILD.md]: include that if you're building outside of a package you should also use /usr/local

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by glowiak** [comment]
Could you accept my previous pr about Slackware, FreeBSD and OpenBSD packages?

**Review by ghost — APPROVED** [review]

**Comment by LennyMcLennington** [comment]
@glowiak have you tested this on OpenBSD? If you've tested it and made sure everything works then I can merge your PRs.

**Comment by glowiak** [comment]
Yeah, but on 32-bit only, I don't have (yet) any 64-bit OBSD VMs, because now I'm busy at updating software on RedStar OS

**Review by d-513 — APPROVED** [review]

**Review by txtsd — APPROVED** [review]
LGTM!

Although this and #131 could've been a single PR.

**Pull request merged**